api/types/authentication.go

```diff
@@ -308,13 +307,13 @@ func (u *U2F) Check() error {
 
 // NewMFADevice creates a new MFADevice with the given name. Caller must set
 // the Device field in the returned MFADevice.
-func NewMFADevice(name string, addedAt time.Time) *MFADevice {
+func NewMFADevice(name, id string, addedAt time.Time) *MFADevice {
```

Comment:
@awly please take a quick look at this small refactor to remove the uuid dependency.

Reply:
LGTM